src/bin/pg_dump/pg_dump.c

```diff
@@ -2223,6 +2241,19 @@ dumpTableData_insert(Archive *fout, const void *dcontext)
 						appendPQExpBufferStr(insertStmt,
 											 fmtId(PQfname(res, field)));
 					}
+
+					/*
+					 * Replace 'rolname' column we added earlier with 'orig_loginname' in
+					 * insert statement's column list.
+					 */
+					if (!found && isBabelfishDatabase(fout) && tbinfo->dobj.namespace &&
+						pg_strcasecmp(tbinfo->dobj.namespace->dobj.name, "sys") == 0 &&
+						pg_strcasecmp(tbinfo->dobj.name, "babelfish_authid_login_ext") == 0)
```

Comment:
Can we simplify this if-condition like `if (!found && is_babelfish_authid_login_ext)`?